src/child/components/Confirmation.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+
+const Confirmation = props => (
+    <div className="confirmation-backdrop" onClick={props.onBackdropClick}>
+        <div className="bubble-head" style={{ top: props.bubbleHeadTop() }}></div>
+        <div className={`confirmation-modal${props.isFlipped ? ' flipped' : ''}`} style={{ top: props.modalTop() }}>
+            <div className="confirmation-content" onClick={e => e.stopPropagation()}>
```

Comment:
I'd probably add this to the bindings if you're taking that approach so we're not declaring the functions in line